core/trino-main/src/main/java/io/trino/sql/planner/iterative/rule/UnwrapCastInComparison.java

```diff
@@ -22,8 +22,10 @@
 import io.trino.spi.TrinoException;
 import io.trino.spi.function.InvocationConvention;
 import io.trino.spi.type.CharType;
+import io.trino.spi.type.DateType;
```

Comment:
the commit title

> Unwrap casts in temporal range comparison expressions

-- can it include "between" somewhere in the title?

perhaps "Unwrap casts in BETWEEN predicate"